pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -43,18 +44,20 @@ type ACMEIssuer struct {
 
 	// PrivateKey is the name of a secret containing the private key for this
 	// user account.
+	// +kubebuilder:validation:Required
```

Comment:
This change doesn't seem to do anything - fields are already considered `required` if they are not `optional`